src/modules/actions.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+import {
+  movieList, displayLikes, getMovieComments, postComment,
+} from './apis.js';
+
+const likesURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const appID = 'wwWCSD2YuNcNPNHza6Cc';
+
+// Counter
+const counter = (arr) => arr.length;
+const commentsCounter = (element) => element.childElementCount;
+
+let truncated = [];
+// Display movies
+const displayMovies = async (shows) => {
+  const listOfShows = document.querySelector('.show-list');
+
+  shows = await movieList();
+  truncated = shows.slice(0, 9);
+
+  truncated.forEach(async (item) => {
+    const show = `
+        <article class="show-card">
+          <div class="img-box">
+            <img class="img" src="${item.image.medium}">
+          </div>
+          <div class="show-detail">
+            <div class="show-detail-top">
+              <h2 class="show-name">${item.name}</h2>                
+              <i data-id="${item.id}" class="fas fa-heart"></i>                
+            </div>
+            <div class="show-detail-bottom">
+                <div class="rate"> 
+                  <p><i class="fas fa-star"></i><span class="rating">${item.rating.average}</span></p>
+                </div>
+                <p>
+                  <span class="likes-count">${await Promise.resolve(displayLikes(item.id))}</span>
+                  <span class="likes">Likes</span>
+                </p>
+            </div>          
+          </div>
+          <button class="comments-btn" id="${item.id}">Comments</button>
+        </article>      
+      `;
+
+    listOfShows.innerHTML += show;
+
+    const likeBtn = document.querySelectorAll('.fa-heart');
+
+    for (let i = 0; i < likeBtn.length; i += 1) {
+      likeBtn[i].addEventListener('click', async (e) => {
+        const likesCount = document.querySelectorAll('.likes-count');
+
+        const likes = parseInt(likesCount[i].textContent, 10) + 1;
+        likesCount[i].textContent = likes;
+
+        await fetch(`${likesURL}${appID}/likes/`, {
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({ item_id: e.currentTarget.dataset.id }),
+        });
+      });
+    }
+    displayDetails();
+  });
+  const pageItems = document.querySelector('.home-count');
+  pageItems.textContent = `${counter(truncated)}`;
+};
+
+const displayDetails = () => {
+  const commentButtons = document.querySelectorAll('.comments-btn');
+  commentButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const projectDetails = document.querySelector('.popup-detail');
+      const main = document.querySelector('.major');
+      projectDetails.classList.add('showpopup');
+      main.classList.add('blur');
+      // console.log(button.id);
```

Comment:
- [ ] Please remove the commented out line to keep the whole code clean and profession